extensions/interactions/EndExploration/directives/EndExploration.js

```diff
@@ -100,15 +100,19 @@ oppia.directive('oppiaResponseEndExploration', [function() {
   return {
     restrict: 'E',
     scope: {},
+    /* eslint-disable angular/no-inline-template */
     template: ''
+    /* eslint-enable angular/no-inline-template */
```

Comment:
Lines 103 and 105 can be replaced by `/* eslint-disable-line angular/no-inline-template */` on line 104. Ditto for the other files.

Reply:
Hi @vojtechjelinek , Thanks!
Setting "allowSimple" to true seems reasonable. Will update the files accordingly.